api_tests/nodes/views/test_node_list.py

```diff
@@ -249,6 +249,19 @@ def test_filtering_tags(self):
         assert_in(self.project_one._id, ids)
         assert_not_in(self.project_two._id, ids)
 
+    def test_filter_no_tags(self):
```

Comment:
I'm not one to complain about improved test coverage, but this seems out of scope. Are preprints identified by tags in the same way that meetings are, or something similar?